frontend/src/app/shared/community-agreement/community-agreement.widget.html

```diff
@@ -0,0 +1,196 @@
+<mat-card class="community-agreement-dialog" #dialogContainer>
+  <mat-card-content class="content-container">
+    <p *ngIf="has_user_agreed">
```

Comment:
NIT: Maybe remove this text for `has_user_agreed`  and update  the `!has_user_agreed`  <p> with text that says something like "Please read and accept the community agreement below to continue to the CSXL web page." Make sure to align it with the rest of the text in the card though!

Reply:
Fixed!